### PR TITLE
Delete developer notes

### PR DESCRIPTION
**Pull request by petervdonovan**
Deletes `developer-notes.md` and resurrects `uf.py`.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,3 +32,19 @@ Please keep your PRs manageable and easy to review.
...
+## Suggested debugging workflow
+For development purposes, it is possible to manually perform an incremental build simply by bypassing Maven and Gradle entirely and
+instead running the Python script `./uf.py`. This script will re-compile Java and Kotlin files and add them to the fat jar using
+the `jar` command with the `-uf` flag.
+
+A suggested workflow is as follows:
+1. Ensure that the appropriate compiler is on your PATH.
+  * To build Java files, `javac` is required.
+  * To build Kotlin files, [the Kotlin JVM compiler](https://github.com/JetBrains/kotlin/releases/tag/v1.5.30) `kotlinc` is required. It must be the JVM compiler, not the native compiler.
+2. Ensure that the language and diagram server fat JAR exists. This file is called `./lib/lflang-lds.jar`. If it does not exist, then it is necessary to build it using the build task: `npm install`.
```
Comment (lhstrh) [inline CONTRIBUTING.md:45]:
```suggestion
2. Ensure that the language and diagram server fat JAR exists. This file is called `./lib/lflang-lds.jar`. If it does not exist, then it is necessary to build it using the build task: `npm run build`.
```

File: uf.py [hunk 2]
```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env python3
+
+"""This script updates individual class files in a language
+and diagram server fat jar that has already been created.
+
+FIXME: Ideally this should be a TypeScript file written in a
+style consistent with build-lds.ts.
+"""
```
Comment (lhstrh) [inline uf.py:8]: Please add author information.

**Review by lhstrh — APPROVED** [review]
Looks good! Only minor comments...

**Review by lhstrh — APPROVED** [review]

File: uf.py (hunk 2 above)
Comment (petervdonovan) [reply uf.py:5]: Done!

**Pull request merged**